python/cuml/test/test_preprocessing.py

```diff
@@ -386,7 +396,8 @@ def test_add_dummy_feature_sparse(sparse_clf_dataset, value):  # noqa: F811
     X_np, X = sparse_clf_dataset
 
     t_X = cu_add_dummy_feature(X, value=value)
-    assert type(t_X) == type(X)
+    #  assert type(t_X) == type(X)
```

Comment:
Do we only test this with CSR? Seems like it supports COO too above?

Reply:
I'll add COO testing.